x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_overview.scss

```diff
@@ -12,12 +12,3 @@
     padding: $euiSizeXL;
   }
 }
```

Comment:
[not a change request, just me thinking out loud] I wonder if I should just get rid of this whole SCSS file now haha 🤔 I added a lot of extra CSS I definitely didn't need to in the beginning..

Reply:
https://github.com/elastic/kibana/pull/97875/commits/603f126e61bc29e776a4b15fe8efac9dd521dbaa